go.mod

```diff
@@ -9,15 +9,15 @@ require (
 	github.com/onsi/ginkgo/v2 v2.1.4
 	github.com/onsi/gomega v1.19.0
 	github.com/ory/dockertest/v3 v3.9.1
-	github.com/rancher/lasso v0.0.0-20220519004610-700f167d8324
+	github.com/rancher/lasso v0.0.0-20221227210133-6ea88ca2fbcc
 	github.com/rancher/wrangler v1.0.1-0.20220520195731-8eeded9bae2a
```

Comment:
Can we bump the wrangler version directly in here?

Reply:
done